src/main/resources/csar/scripts/kubectl_statefulset_scale.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# configuration
+KUBE_ADMIN_CONFIG_PATH=/etc/kubernetes/admin.conf
+
+# provided variables:
+# KUBE_STATEFULSET_ID: contains the k8s statefulset id to undeploy
+# EXPECTED_INSTANCES: contains the replicas count for the statefulset
+
+NAMESPACE_OPTION=""
+if [ ! -z "$NAMESPACE" ]; then
+   NAMESPACE_OPTION="-n $NAMESPACE "
+fi
+
+function scale_resource(){
+    # scale to new expected instances count
+    kubectl --kubeconfig "${KUBE_ADMIN_CONFIG_PATH}" ${NAMESPACE_OPTION}scale statefulset "${KUBE_STATEFULSET_ID}" --replicas="${EXPECTED_INSTANCES}"
```

Comment:
```suggestion
    kubectl --kubeconfig "${KUBE_ADMIN_CONFIG_PATH}" "${NAMESPACE_OPTION}scale" statefulset "${KUBE_STATEFULSET_ID}" --replicas="${EXPECTED_INSTANCES}"
```

Reply:
In fact in jus did a copy paste of deployment script :no_mouth:   
I replaced it with void.sh in the tosca file instead